kano_i18n/assets.py

```diff
@@ -0,0 +1,19 @@
+# assets.py
+#
+# Copyright (C) 2015 Kano Computing Ltd.
+# License: http://www.gnu.org/licenses/gpl-2.0.txt GNU GPL v2
+#
+# Utility function for finding non-text translated assets
+#
+
+import locale
+import os
+
+
+def get_path(base, relative, default='en_US'):
```

Comment:
I would add a docstring to this explaining where i18n assets would have to be installed, params, etc

Reply:
Done.